### DISCUSSION TITLE
Is it possible to extract the PDE equations or ODE-description for a given model?

### DISCUSSION THREAD

**Discussion by AverageDaniel** [pybamm-team/PyBaMM#3748] [Q&A] [answered]
Hello,
I am a new user of PyBaMM and was just wondering if it is possible to somehow extract information the set of equations that the solver is working on for the given model setup?

**Comment by TomTranter** [comment, marked as answer]
Yes you can use latexify https://docs.pybamm.org/en/stable/source/examples/notebooks/models/latexify.html

**Reply by AverageDaniel** [reply]
Thanks a lot
